dts/armbian/detect_board.inc

```diff
@@ -72,6 +72,12 @@ if [ ! -z "$DEV_TREE_COMPATIBLE" ]; then
         INCLUDE_FILE='/var/lib/piVCCU/dts/nanopim4.dts.include'
         break
         ;;
+     radxa,rockpi-4b)
```

Comment:
Needs to be "radxa,rockpi4", as this is the name in the original DTS: https://github.com/torvalds/linux/blob/788a024921c48985939f8241c1ff862a7374d8f9/arch/arm64/boot/dts/rockchip/rk3399-rock-pi-4.dts#L15

Reply:
Good catch, I was testing with armbian which seems to have an older version of that patch. https://github.com/armbian/build/blob/master/patch/kernel/rockchip64-dev/add-board-rockpi4b.patch#L33

Reply:
ah, ok. In that case, both names should be included

Reply:
I've sent a fix to armbian https://github.com/armbian/build/pull/1399 so only `radxa,rockpi4` needs to be included

Reply:
Armbian has updated their kernels now, I'll test it as soon as I get the opportunity to.